apps/web/src/components/search/index.js

```diff
@@ -20,16 +20,29 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import * as Icon from "../icons";
 import "./search.css";
 import Field from "../field";
+import { useStore } from "../../stores/theme-store";
+import { getColors } from "@notesnook/theme/dist/theme/colorscheme";
 
 function SearchBox(props) {
+  const { theme } = useStore();
+  const { text } = getColors(theme);
```

Comment:
You don't need to use `getColors` to get theme colors. We are using `theme-ui` internally so stuff like colors, dimensions, margins etc are automatically resolved by it.